lib/internal/modules/esm/load.js

```diff
@@ -29,13 +29,12 @@ const {
  * @param {ESModuleContext} context used to decorate error messages
  * @returns {Promise<{ responseURL: string, source: string | BufferView }>}
  */
-async function getSource(url, context) {
+function getSource(url, context) {
```

Comment:
This function is now functionally equivalent to `getSourceSync`.
Should it be removed in favor of `getSourceSync`, since that one is exported while `getSource` is not?

Reply:
Oow! Likely yes 🙌

Reply:
It’s actually identical to `getSourceSync`.